addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -160,4 +175,78 @@ options.registry.website_sale = options.Class.extend({
         }).then(this.reload);
     }
 });
+
+/**
+ * Handles the edition of snippet's products search bar.
+ */
+options.registry.productsSearchBar = options.Class.extend({
```

Comment:
I know it is not the convention right now but it should: start the option name with an uppercase later, as it is a class (adapt data-js="..." accordingly).